image/config.in

```diff
@@ -13,9 +13,11 @@ mainmenu_option next_comment
 
 	if [ "$CONFIG_IMG_MINIX" == "y" -o "$CONFIG_IMG_ROM" == "y" ]; then
 		define_bool CONFIG_IMG_LINK y
-		fi
+	fi
 
-	if [ "$CONFIG_IMG_ROM" != "y" ]; then
+	if [ "$CONFIG_IMG_RAW" == "y" ]; then
+		:
+	elif [ "$CONFIG_IMG_ROM" != "y" ]; then
```

Comment:
See previous comment. Still have to select the image format even if no filesystem.

Reply:
Ok. So the TARGET_BLKS will be used to create the image size, correct?

Reply:
Correct.